src/runmode-dpdk.c

```diff
@@ -1216,6 +1233,10 @@ static void DeviceInitPortConf(const DPDKIfaceConfig *iconf,
     if (dev_info->tx_offload_capa & RTE_ETH_TX_OFFLOAD_MBUF_FAST_FREE) {
         port_conf->txmode.offloads |= RTE_ETH_TX_OFFLOAD_MBUF_FAST_FREE;
     }
+
+    if ((dev_info->rx_offload_capa & RTE_ETH_RX_OFFLOAD_VLAN_STRIP) && iconf->vlan_strip_enabled) {
+        port_conf->rxmode.offloads |= RTE_ETH_RX_OFFLOAD_VLAN_STRIP;
```

Comment:
can we add a `SCLogConfig` message here to indicate the vlan stripping has been enabled for this interface?

Reply:
also wondering if we should issue a warning if the setting is enabled, but the iface doesn't support it

Reply:
Sure, I can update that.